src/main/java/coydir/ui/PersonInfo.java

```diff
@@ -54,6 +58,8 @@ public void update(Person person) {
         phone.setText("Phone number:  " + person.getPhone().value);
         address.setText("Address:  " + person.getAddress().value);
         email.setText("Email address:  " + person.getEmail().value);
+        totalLeave.setText("Total Leave: " + person.getTotalNumberOfLeaves());
```

Comment:
This one not really important, since it's probably going to be revamped later on, but can add a 's' so it's "Total Leaves"?

Reply:
Done!